src/main/java/dev/meyi/bn/modules/calc/CraftingCalculator.java

```
@@ -116,7 +111,7 @@ public static ArrayList<ArrayList<String>> getBestEnchantRecipes() {
             list.add(new ArrayList<>(Arrays
                 .asList(String.valueOf(profitInstaSell), Double.toString(profitSellOffer),
                     Double.toString(pricePerMil), itemName)));
-          } catch (JSONException ignored) {
+          } catch (Exception ignored) {

```

Let's not swallow exceptions - again, only catch what's needed and then `ignored.printStackTrace()`